### ISSUE TITLE
Deleteting a MobilityHindrance doesn't remove it from hasConsequences of the Groundwork it was initially created for

### ISSUE THREAD

**Issue by ireactsam** [Informatievlaanderen/GIPOD#258]
- GET "https://private-api.gipod.vlaanderen.be/api/v1/groundworks/9144122"
```
  ...
  "hasConsequence": [
    {
      "@id": "/api/v1/mobility-hindrances/11205229",
      "@type": "MobilityHindrance",
      "gipodId": 11205229
    },
    {
      "@id": "/api/v1/mobility-hindrances/11205230",
      "@type": "MobilityHindrance",
      "gipodId": 11205230
    }
  ],
  ...
```

- DELETE "https://private-api.gipod.vlaanderen.be/api/v1/mobility-hindrances/11205230" (x-correlation-id: 2d335b36-6f86-46e0-a262-963b796391e8)
- GET "https://private-api.gipod.vlaanderen.be/api/v1/mobility-hindrances/11205230" (x-correlation-id: a9879eb0-71d0-4a85-84bf-299b88b5df57) --> 410 Gone
- GET "https://private-api.gipod.vlaanderen.be/api/v1/groundworks/9144122" -->  SAME RESULT

Expected: 11205230 is no longer in the `hasConsequence` list

**Comment by AIV-Bert** [comment]
Hi @ireactsam,

This is indeed a bug and not the expected behavior.
We are aware of this issue and this was added to our backlog. 

Thank you for reporting this.

Kind regards,

Bert

**Comment by AIV-Bert** [comment]
Hi @ireactsam,

This issue wil be resolved in the 1.4.0 version of our api. Existing data will be corrected.

Kind regards,

Bert

**Comment by AIV-Bert** [comment]
Released on production
